source/before-integrating/index.html.md.erb

```diff
@@ -0,0 +1,60 @@
+---
+title: Before you integrate with GOV.UK One Login
+weight: 2.0
+last_reviewed_on: 2023-09-15
+review_in: 6 months
+---
+
+# Before you integrate with GOV.UK One Login
+
+Before you begin your integration with GOV.UK One Login, you should consider:
+
+* how many services within your organisation you’re planning to integrate
+* if your services need to share users, in case you’re integrating more than 1 service
+* if you need to create a reusable component to standardise integration across your organisation, in case you’re integrating a large number of services
+* what the scope of your individual services is and whether this meets the GOV.UK Service Standard definition of a service 
+
+If you’re integrating multiple services, you should [create a configuration for each service you’re integrating](/before-integrating/#create-a-configuration-for-each-service-you-re-integrating). 
+
+Make sure you scope your services according to the [GOV.UK Service Standard guidance](https://www.gov.uk/service-manual/service-standard) on how users think and what they need to do. Find more [information on scoping your service](https://www.gov.uk/service-manual/design/scoping-your-service). 
+
+## Create a configuration for each service you’re integrating
+
+GOV.UK One Login is an OpenID Connect (OIDC) provider. An OIDC ‘relying party’ is an app that outsources its user authentication function to an identity provider, which in this instance is GOV.UK One Login.
+
+To interact with GOV.UK One Login, you must first [register each of your services with GOV.UK One Login as a relying party](/integrate-with-integration-environment/manage-your-service-s-configuration/#register-your-service-to-use-gov-uk-one-login). You need to do this for each of the services that you want to integrate with GOV.UK One Login.
+
+Part of the service’s configuration is the `client-id`, which is a unique identifier that GOV.UK One Login uses to identify your services. Each service should have a distinct `client-id`.
+
+GOV.UK One Login uses `client-id` for: 
+
+* retrieving configurations
+* event auditing
+* performance analytics 
+* fraud prevention and data sharing
+
+When you are ready to set up your service’s configuration, follow the guidance on how to [manage your service’s configuration with GOV.UK One Login](/integrate-with-integration-environment/manage-your-service-s-configuration/#manage-your-service-s-configuration-with-gov-uk-one-login).
+
+### Why you should use a specific configuration for each service
+
+You must use individual configurations for each of your services to get the following benefits: 
+
+* GOV.UK One Login can provide a report detailing each service’s performance including success rates and volumes
+* if a service gets compromised, the outage is limited to an individual service
```

Comment:
```suggestion
* in the event of a service outage, the effect is limited to a single service - your other service will not be affected
```

Reply:
Addressed in the following commit (https://github.com/govuk-one-login/authentication-tech-docs/pull/133/commits/a5a596c29d77fb8c3719ad7a1fd8f0c108363d32).